rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 
 <collection-ic> a skos:Collection ;
```

Comment:
I think... yes and yes. Both of these are handled in the versions PR :smile_cat: